pkg/topology/query.go

```diff
@@ -198,6 +198,33 @@ func (p TopologyParams) GetComponentRelationWhereClause() string {
 	return s
 }
 
+func QueryRenderComponents(ctx context.Context, systemTemplateID string) ([]pkg.Renderers, error) {
+	query := `select spec->'renderers' from templates where id = $1`
+	args := map[string]interface{}{
+		"id": systemTemplateID,
+	}
+
+	rows, err := db.QueryNamed(ctx, query, args)
```

Comment:
Don't use `QueryNamed` function, prefer using Gorm or the underlying pool object to query